Documentation/contributing/development/reviewers_committers/review_process.rst

```diff
@@ -98,6 +98,25 @@ Review process
    | ``upgrade-impact``       | The code changes have a potential upgrade impact                          |
    +--------------------------+---------------------------------------------------------------------------+
 
+#. When submitting a review, provide explicit approval or request for changes
+   whenever possible. This provides clear feedback to contributors about
+   whether they must take action to resolve the comments before the PR can be
+   merged.
+
+   If you are in doubt regarding the comments or review, you may leave comments
+   without explicit approval. If you do not explicitly approve or request
+   changes, it's best practice to raise awareness about the discussion so that
+   others can participate and clarify expectations to the contributor. Here are
+   some ways that you can raise awareness:
```

Comment:
```suggestion
   If you need more information before you can approve or request changes, you can leave comments
   seeking clarity. If you do not explicitly approve or request
   changes, it's best practice to raise awareness about the discussion so that
   others can participate. Here are some ways you can raise awareness:
```